### ISSUE TITLE
[BUG] ProtocolLib 4.5.1 no longer works on latest 1.12 Magma release.

### ISSUE THREAD

**Issue by VOrlando520** [magmafoundation/Magma#469]
**Describe the bug**
A clear and concise description of what the bug is: Latest 1.12 magma breaks any support for ProtocolLib 4.5.1+ (latest known PLib version to work with Magma) This in turn breaks a ton of plugins that rely on ProtocolLib

**To Reproduce**
Install ProtocolLib 4.5.1
Install needed plugin that requires PLib (ala FastLogin, anything along those lines)
Boot up server and it disables PLib and in turns disables any plugins reliant on it.

**Expected behavior**
A clear and concise description of what you expected to happen:
What was expect is for it to boot PLib up with the latest reported working version from the Magma team.

**Screenshots**
If applicable, add screenshots to help explain your problem:
Upon server boot it disables ProtocolLib and throws this error
https://pastebin.com/44iEFqkc

**Info (please complete the following information):**
- Mod Name: ProtocolLib
 - Mod Version: v4.5.1
 - Magma Version: Latest Magma 1.12 release
 - OS:

**Additional context**
Add any other context about the problem here:

**Comment by VOrlando520** [comment]
I dont believe they are focusing too much on 1.12 anymore, which is sad but ya know. Was hoping they would at least look at this because it breaks a ton of plugins that rely on PLib (orebfuscator, fastlogin, etc.)

**Comment by zoyer2** [comment]
yep same
